typings/pencil-code-embed-defs.d.ts

```diff
@@ -0,0 +1,88 @@
+// Code - https://pencilcode.net/lib/pencilcodeembed.js
+// As only one class is exported in the library, so type definitions
```

Comment:
qq: Does exported in the library means only class used in our codebase.

Rephrase: Type definitions are only added for PencilCodeEmbed class since it is the only class used in our Codebase.

Reply:
Done!

Reply:
Does exported in the library means only class used in our codebase?

Can you please answer this as well?

Reply:
If you see at the end of library code
```javascript
if (global.define && global.define.amd) {
    // Support AMD-style loading, if present.
    define(function() {
      return {
        PencilCodeEmbed: PencilCodeEmbed
      };
    });
  } else {
    // Otherwise, just set the global symbol.
    global.PencilCodeEmbed = PencilCodeEmbed
  }
```
Only this class is returned I mean other functions like

```javascript
function makeSecret() {
    var SECRET_LENGTH = 64;

    var secret = '';
    for (var i = 0; i < SECRET_LENGTH; i++) {
      secret += String.fromCharCode(65 + Math.floor(Math.random() * 26));
    }

    return secret;
  }
```

are not returned. So, we don't need add type definitions for those.

Reply:
Ok, got your point. Thanks for the clarification. 

In that case can you please update the comment to:

```
Type definitions are only added for PencilCodeEmbed class since it is the only class exported by the library.
```

Reply:
Done!